client/accounting.go

```diff
@@ -46,17 +46,17 @@ func (x ResBalanceGet) Amount() accounting.Decimal {
 // NeoFS status codes are returned as `error`, otherwise, are included
 // in the returned result structure.
 //
-// Immediately panics if parameters are set incorrectly (see PrmBalanceGet docs).
 // Context is required and must not be nil. It is used for network communication.
 //
+// Return errors:
+//   - [ErrMissingAccount]
```

Comment:
what should the user do with them? it seems like nothing. lets leave them unexported for now

Reply:
Use should check it with `errors.Is` as any another errors which SDK may return if user needs it. It will be completely useless if it unreported. We don't need separate error for this situation. How I understand, we adding this errors more for SDK users/clients rather than for us

Reply:
ofc we do it for users. User call the method, and
```
if errors.Is(client.ErrMissingAccount) {
  // then what?
}
```
Account is required parameter, user must set it before the call. Panic told him that he made a gross mistake in the development, errors can be just rendered

Reply:
I don't know good example to cover this. But I think it is a good point to give users some chance to understand what certain error occurred. This error has no difference with any another error. It's just a way to say - something went wrong.

I had few situations when lib developer made separate error for some situation, but left it unexported and I had to find another complicated ways to check the error.

Reply:
we have exported errors which allow user to tune behavior of its app in terms of NeoFS: `ObjectNotFound`, `AccessDenied`, etc. Here is just an input error useful only as text, if user sees it he fixes his code and the error won't happen ever again.

Ofc we can't predict everything. if suddenly someone needs this error, we could export it and it will be clearly justified and backwards compatible.

Reply:
I understand you worry. We may have a meeting and talk about such errors. I mean should it be exported or not. In this issue we just replace panics with errors and I suggest left them all exported, for now

Reply:
> left them all exported, for now

don't forget that we can't export them now and unexport later until next major release (v2)

Reply:
I'd say the way we have this API now they better be exported (but this may change after #412). In general, good (ok, perfect ones) APIs list _all_ of their potential errors explicitly with some notes on what could be done if you're receiving one of them.

Reply:
> what could be done if you're receiving one of them

agree, but with one remark: what can be done _programmatically_

after #412 almost all introduced errors will become non-actual anymore

Reply:
lets leave exported, one more rc is planned anyway